.gitignore

```diff
@@ -27,7 +27,6 @@ var/
 #  Usually these files are written by a python script from a template
 #  before PyInstaller builds the exe, so as to inject date/other infos into it.
 *.manifest
-*.spec
```

Comment:
I tested the build on Linux, everything works fine now. 

PyInstaller doesn't work with Python 3.6 yet but support of 3.6 is coming in the next release. See https://github.com/pyinstaller/pyinstaller/issues/2331